WebUI/generate.sh

```diff
@@ -41,9 +42,9 @@ BUILD_DIR=$(pwd)/yaydoctemp
 
 cd $ROOT_DIR
 
-cp -a ../../../scripts/ $BUILD_DIR/
-cp -a ../../../templates/ $BUILD_DIR/
-cp -a ../../../requirements.txt $BUILD_DIR/
+cp -a ../../../../scripts/ $BUILD_DIR/
+cp -a ../../../../templates/ $BUILD_DIR/
+cp -a ../../../../requirements.txt $BUILD_DIR/
```

Comment:
This change was introduced by you in the last PR. So you're saying that you made the changes and sent a PR without even testing it?